tests/dags/database/test_report_pending_reported_media.py

```diff
@@ -0,0 +1,60 @@
+from unittest import mock
+
+import pytest
+from database.report_pending_reported_media import report_actionable_records
+
+
+def _make_reported_records_data(media_type: str):
+    return [
+        (
+            {"dcma": 1, "mature": 2, "other": 3},
+            [
+                f"{media_type}: 6",
+                "_(1 dcma, 2 mature, 3 other)_",
+            ],
+        ),
+        (
+            {"dcma": 1, "mature": 2},
+            [
+                f"{media_type}: 3",
+                "_(1 dcma, 2 mature)_",
+            ],
```

Comment:
I always make this mistake too :sweat_smile: 

```suggestion
            {"dmca": 1, "mature": 2, "other": 3},
            [
                f"{media_type}: 6",
                "_(1 dmca, 2 mature, 3 other)_",
            ],
        ),
        (
            {"dmca": 1, "mature": 2},
            [
                f"{media_type}: 3",
                "_(1 dmca, 2 mature)_",
            ],
```

Reply:
WOW same 😂

Reply:
Ha would you believe that I thought I checked for this typo? Almost as bad as trying to type 'csv' 😆